pkg/service/system/route53.go

```diff
@@ -40,6 +43,8 @@ var regionClassicLoadBalancerHostedID = map[string]string{
 	"us-gov-west-1":  "Z33AYJ8TM3BH4J",
 }
 
+const defaultRegion = "us-west-1"
```

Comment:
if the use case is to create a session, then don't use the default region, pass empty string.

Reply:
created a constant "emptyRegion"